schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml

```diff
@@ -280,10 +280,10 @@
       <summary>Isolate volumes, devices and trash windows</summary>
       <description>Consider volume, devices and trash as different application windows and not part of the file manager</description>
     </key>
-    <key type="b" name="dance-urgent-applications">
+    <key type="b" name="wiggle-urgent-applications">
```

Comment:
Please, don't change this, otherwise people's setting won't be changed.

It's fine to have it called differently here.

Reply:
Reverted.